tests/channel_ut/channel_ut.c

```diff
@@ -326,7 +361,7 @@ TEST_FUNCTION(channel_push_fails_with_null_out_op_push)
     //arrange
 
     //act
-    CHANNEL_RESULT result = channel_push(test_channel, test_data_rc_ptr, test_push_callback, test_push_context, NULL);
+    CHANNEL_RESULT result = channel_push(test_channel, g.g_correlation_id, test_data_rc_ptr, test_push_callback, test_push_context, NULL);
```

Comment:
>g_correlation_id [](http://example.com/codeflow?start=57&length=16)

let's use separate correlation_ids for push and pull and see what happens.

Reply:
Done